lib/Config.php

```diff
@@ -76,12 +89,12 @@ public function getStore()
 
     public function getIdAttribute()
     {
-        return $this->config->getString(self::USER_ID_ATTRIBUTE, 'uid');
+        return $this->idAttribute;
     }
 
-    public function getSideInfo($side)
+    public function getSideInfo(string $side)
     {
-        assert(in_array($side, [self::SIDES], true));
+        assert(in_array($side, self::SIDES, true));
```

Comment:
`assert` really does nothing, so they can all be removed. If it needs to be checked, there needs to be an `if` and `throw`.